lesson-1/package.json

```diff
@@ -3,17 +3,29 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "apollo-boost": "^0.1.22",
-    "graphql": "^14.0.2",
-    "react": "^16.4.2",
-    "react-apollo": "^2.3.1",
-    "react-dom": "^16.4.2",
-    "react-scripts": "1.1.4"
+    "apollo-boost": "^0.4.9",
+    "graphql": "^15.5.0",
+    "react": "^17.0.1",
+    "react-apollo": "^3.1.5",
+    "react-dom": "^17.0.1",
+    "react-scripts": "4.0.3"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
```

Comment:
Running `yarn start` would give me this prompt: 
```
We're unable to detect target browsers.

Would you like to add the defaults to your package.json? › (Y/n)
```

Hitting `Y` adds this chunk of code to the package.json.